ts/features/walletV3/onboarding/types/index.ts

```diff
@@ -22,15 +42,15 @@ export type OnboardingSuccess = {
  */
 export type OnboardingFailure = {
```

Comment:
Could we make the scope of this type clearer by adding more information in the name? I think `OnboardingWebViewFailure` could be more clear about the source of this type of error.

Reply:
What is unclear about the current type name? Because right now there are three types of outcomes available:

- `OutcomeSuccess`: which represents a successful outcome and therefore the paper was onboarded correctly
- `OutcomeFailure`: represents a failed onboarding outcome that is generated in the web-side onboarding process
- `OutcomeError`: represents an internal our outcome that is generated exclusively when the WebView of the app for any kind of problem does not load properly or goes into an error in rendering the onboarding page

Since these are the only three possible outcomes, following your suggestion it would make more sense to put it at the error outcome (OutcomeError => `OutcomeWebViewError`) instead of at the failure outcome, what do you think?